packages/form-builder/addon/components/cfb-slug-input.js

```diff
@@ -30,6 +31,12 @@ export default class CfbSlugInputComponent extends Component {
 
   @action
   calculatePadding(element) {
+    // Ensures that the element is already visible in combined use
+    // with modal dialogs.
+    scheduleOnce("afterRender", this, "_calculatePadding", element);
+  }
+
+  _calculatePadding(element) {
```

Comment:
BTW just some random JS facts: if you really want this to be private, use `#calculatePadding`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties

Reply:
> BTW just some random JS facts: if you really want this to be private, use `#calculatePadding`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties

Thanks. :santa: 

`scheduleOnce` can not resolve private methods provided as string reference. But we could provide it directly as a function. But then again your previous critic comes into effect. ("*Must be a string, otherwise context binding will be weird*"). I propose we stick with the pseudo private underscore oldsql naming.

Reply:
TIL